test/invariant/inferences_test.go

```diff
@@ -15,8 +15,8 @@ import (
 
 func doInferenceAndReputation(
 	m *testcommon.TestConfig,
-	leaderWorker Actor,
-	leaderReputer Actor,
+	_ Actor,
```

Comment:
why are these params needed to be passed in at all if they are not used? Can't we remove these params?

Reply:
no they're part of an interface that is expected by the "list of actions a fuzz round can take"

some functions don't use them, some do, the ones that don't just ignore em